packages/orchestration/test/test-withdraw-reward.js

```diff
@@ -31,7 +35,17 @@ const scenario1 = {
   delegations: {
     agoric1valoper234: { denom: 'uatom', amount: '200' },
   },
-};
+});
+const scenario2 = /** @type {const} */ ({
```

Comment:
`scenario1` and `scenario2` are symptomatic of naming fatigue. Perhaps I should change to `scenarioStaking` and `scenarioRedelgation`.